src/js/common/components/AlertBox.jsx

```diff
@@ -52,13 +51,33 @@ class AlertBox extends React.Component {
       );
     }
 
+    if (this.props.headline) {
+      return (
+        <div
+          aria-live="assertive"
+          className={alertClass}
+          ref={(ref) => { this._ref = ref; }}>
+          <div className="usa-alert-body">
+            <div className="usa-alert-heading">
+              {this.props.headline}
+            </div>
+            <div className="usa-alert-text">
+              {this.props.content}
+            </div>
+          </div>
+          {closeButton}
+          <div className="cf"></div>
+        </div>
+      );
+    }
+
```

Comment:
Instead of returning a whole different chunk of code by looking at `this.props.headline`, you can assign to a variable like:

```jsx
let headline = this.props.headline ? (<div className="usa-alert-heading">{this.props.headling}</div>) : null;
```
Then just insert the `{headline}` in the HTML below and it'll only be shown if not null.